### DISCUSSION TITLE
= sign alignment while each line in numbered

### DISCUSSION THREAD

**Discussion by hamaney** [typst/typst#4140] [Q&A] [answered]
I tried to do this in Typst, but I could not. I am wondering if there is a way to have a number on each line and have the = sign aligned

# The Goal
This is done in LaTeX, and we have to imitate it in Typst.
<img width="1209" alt="image" src="https://github.com/typst/typst/assets/11434409/85f0fec4-633d-4fb8-9e06-9b36960e8ee1">

# What I have

<img width="710" alt="image" src="https://github.com/typst/typst/assets/11434409/8fc0dec4-0e40-44f6-9755-a3f67927db3e">

<img width="695" alt="image" src="https://github.com/typst/typst/assets/11434409/4a63bdfa-f1cd-4b03-a791-a558ab07e4df">

It seems that I can only have one of the two options above but not both.

# DOCS
I am familiar with this docs, but it does not seem to have numbering of each line of equation while keeping = sign aligned 

<img width="1016" alt="image" src="https://github.com/typst/typst/assets/11434409/1d51ed34-26ed-4186-96a7-da58971aa9e2">

**Comment by EpicEricEE** [comment, marked as answer]
Check out the [`equate`](https://typst.app/universe/package/equate) package:

```typ
#import "@preview/equate:0.1.0": equate

#set math.equation(numbering: "(1)")
#show: equate

Possibly we'll need some computation that lies across several lines
$ x^2 + y^2 &= 4 \
        y^2 &= 4 - x^2 \
          y &= sqrt(4 - x^2) $
```

![image](https://github.com/typst/typst/assets/7191192/656d2b72-7f04-433a-9995-f6e2833db6ea)

**Reply by l0wx** [reply]
@EpicEricEE Just the moment I wanted to submit …^^

**Reply by hamaney** [reply]
Greatly appreciating the quick response!
